### PR TITLE
Add ts() function within CRM_Core_Error::statusBounce() method

### PR DESCRIPTION
**Pull request by scardinius**
Overview
----------------------------------------
I couldn't translate *you have tried to access a report that does not exist* text.

Before
----------------------------------------
Some texts within *CRM_Core_Error::statusBounce()* method don't have translation function.

After
----------------------------------------
All *CRM_Core_Error::statusBounce()* methods have translation function `ts()`.

### CODE REVIEW

**Comment by demeritcowboy** [comment]
Intermittent test fail: api.v4.Action.CreateWithOptionGroupTest.testWithCustomDataForMultipleContacts

jenkins retest this please

File: CRM/Contribute/Page/ContributionRecur.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ public function view() {
       ]);
     }
     catch (Exception $e) {
-      CRM_Core_Error::statusBounce('Recurring contribution not found (ID: ' . $this->getEntityId());
+      CRM_Core_Error::statusBounce(ts('Recurring contribution not found (ID: %1', [1 => $this->getEntityId()]));
```
Comment (demeritcowboy) [inline CRM/Contribute/Page/ContributionRecur.php:50]:
```suggestion
      CRM_Core_Error::statusBounce(ts('Recurring contribution not found (ID: %1)', [1 => $this->getEntityId()]));
```

The `)` was missing before but might as well fix it.

File: CRM/Event/Page/ParticipantListing.php [hunk 2]
```diff
@@ -75,7 +75,7 @@ public function run() {
       ) . '.php';
     $error = include_once $classFile;
     if ($error == FALSE) {
-      CRM_Core_Error::statusBounce('Participant listing code file: ' . $classFile . ' does not exist. Please verify your custom particpant listing settings in CiviCRM administrative panel.');
+      CRM_Core_Error::statusBounce(ts('Participant listing code file: %1 does not exist. Please verify your custom particpant listing settings in CiviCRM administrative panel.', [1 => $classFile]));
```
Comment (demeritcowboy) [inline CRM/Event/Page/ParticipantListing.php:78]: I didn't even know this existed! It's a bit technical and seems unlikely to be something a user would deal with - so it's a judgement call I guess whether to include it in translation.

Comment (demeritcowboy) [reply CRM/Event/Page/ParticipantListing.php:78]: Also participant is spelled wrong. (particpant)

Comment (eileenmcnaughton) [reply CRM/Event/Page/ParticipantListing.php]: Good points - I think I've managed to add a spelling fix - the other is a judgement call as you say & I'm on the side of 'getting this merged'

**Pull request merged**